src/views/AddItem.jsx

```diff
@@ -1,3 +1,108 @@
+import { useState, useEffect } from 'react';
+import { addItem } from '../api/firebase';
+
+const dayConverter = (text) => {
+	if (text === 'soon') {
+		return 7;
+	} else if (text === 'kind-of-soon') {
+		return 14;
+	} else {
+		return 30;
+	}
+};
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [itemName, setItemName] = useState('');
+	const [frequency, setFrequency] = useState('soon');
+	const [message, setMessage] = useState('');
+	const listId = 'my test list';
```

Comment:
**Question:** Is there any reason the placeholder `listId` can't be replaced with our global state variable `listToken` at this point?

You mentioned in the PR message that this change will need to be made in the future, so I'm curious if there's a blocker to making the edit within this PR?

Reply:
You are right. For some reason, listToken wasn't "my test list" when we printed it. 
That's why we hardcoded.   

I checked it again and it returned "my test list". We will update this!!

Reply:
Done, we passed the prop and removed the temporary variable.